openedx/core/djangoapps/user_api/accounts/views.py

```diff
@@ -202,3 +205,23 @@ def partial_update(self, request, username):
             )
 
         return Response(account_settings)
+
+
+class AccountDeactivationView(APIView):
+    """
+    Account deactivation viewset. Currently only supports POST requests.
+    Only admins can deactivate accounts.
+    """
+    permission_classes = (permissions.IsAuthenticated, IsStaffUser)
```

Comment:
Do you know where the ``User`` model is located in edx-platform? I'm having trouble finding it.